### PR TITLE
Update to typed-css-modules 0.5.1

### PR DESCRIPTION
**Pull request by gdelmas**
`typed-css-modules` now includes typings and a `loaderPlugins` option.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,44 +1,17 @@
...
 import {Plugin as PostCssPlugin} from 'postcss';
 import {Tapable} from 'tapable';
 import {promisify} from 'util';
 import {Compiler} from 'webpack';
-
-const cssModuleCore = require('css-modules-loader-core');
-const DtsCreator = require('typed-css-modules');
-const FileSystemLoader = require('typed-css-modules/lib/fileSystemLoader').default;
+import cssModuleCore = require('css-modules-loader-core');
+import DtsCreator = require('typed-css-modules');
```
Comment (pyrocat101) [inline src/index.ts]: Why remove the typing here?

File: src/index.ts [hunk 2]
```diff
@@ -1,44 +1,17 @@
...
 import * as glob from 'glob';
 import {Plugin as PostCssPlugin} from 'postcss';
 import {Tapable} from 'tapable';
 import {promisify} from 'util';
 import {Compiler} from 'webpack';
-
-const cssModuleCore = require('css-modules-loader-core');
-const DtsCreator = require('typed-css-modules');
-const FileSystemLoader = require('typed-css-modules/lib/fileSystemLoader').default;
+import cssModuleCore = require('css-modules-loader-core');
```
Comment (pyrocat101) [inline src/index.ts]: can you use `import * as cssModuleCore from 'css-modules-loader-core'`?

File: src/index.ts (hunk 1 above)
Comment (gdelmas) [reply src/index.ts]: as the new `typed-css-modules` version is now written in typescript, it comes with its own typings. therefore there is no more need to define them here scetchyly.

File: src/index.ts (hunk 2 above)
Comment (gdelmas) [reply src/index.ts]: i think there was some issue with that. but let me try again.

Comment (gdelmas) [reply src/index.ts]: it is possible. my bad. i had tried `esModuleInterop` but this would have changed the emit.
unfortunately `DtsCreator` has to be imported with require, as it is defined in CommonJS style `export = ...`.

**Comment by pyrocat101** [comment]
@gdelmas thanks for the update! I think there are still a few places where `import X = require` is used. I will stamp once you change them into the standard ES import syntax.

**Comment by gdelmas** [comment]
@pyrocat101 as i was saying. the import with require is necessary:

> unfortunately DtsCreator has to be imported with require, as it is defined in CommonJS style export = ....

**Comment by pyrocat101** [comment]
Created an issue https://github.com/Quramy/typed-css-modules/issues/76 to track.
